main.js

```diff
@@ -1,5 +1,9 @@
 createLoginForm();
 
+let validationErrors = {};
+let formValues = {};
```

Comment:
`function createLoginForm()` should create form that depends on this variables. 

Example:
You have this line of code `parentElement.appendChild(createInputField("form_username","username", "Username"));`

but I want something like this 
```
parentElement.appendChild(createInputField({
    inputValue: formValues.username,
    error: validationErrors.username,
}));
```